### DISCUSSION TITLE
Complete the expression

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#1145] [Galaxy Trivia] [answered]
> Be careful about 🤓 health 📖...

**Bonus:** Name the author.

**Comment by thuongtruong1009** [comment]
"Be careful about your health, it's the most valuable asset you have." 🤓📖 

The quote "Be careful about your health, it's the most valuable asset you have" does not have a specific author that I can attribute to it. It's a general piece of advice often given by various health and wellness experts.

**Reply by ParanoidUser** [reply]
🔥Would you like to give it another try?

**Comment by tomerf** [comment, marked as answer]
"Be careful about reading health books. You may die of a misprint."
The quote is attributed to Mark Twain.

**Reply by ParanoidUser** [reply]
Bingo! 👏👏
